components/server/src/workspace/headless-log-service.ts

```diff
@@ -124,14 +124,15 @@ export class HeadlessLogService {
      *
      * @param userId
      * @param wsi
+     * @param ownerId
      * @param taskId
      * @returns
      */
-    async getHeadlessLogDownloadUrl(userId: string, wsi: WorkspaceInstance, taskId: string): Promise<string | undefined> {
+    async getHeadlessLogDownloadUrl(userId: string, wsi: WorkspaceInstance, ownerId: string, taskId: string): Promise<string | undefined> {
         try {
             return await new Promise<string>((resolve, reject) => {
                 const req = new LogDownloadURLRequest();
-                req.setOwnerId(userId);
+                req.setOwnerId(ownerId);
```

Comment:
👏🏻  thank you so much!!!

Reply:
:bow:  Sorry for this. Has been introduced with recent proxy endpoint I think.